osu.Game/Overlays/Profile/Header/PreviousUsernamesContainer.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Input.Events;
+using osu.Game.Graphics;
+using osu.Game.Users;
+using osuTK;
+using System;
+using System.Linq;
+
+namespace osu.Game.Overlays.Profile.Header
+{
+    public class PreviousUsernamesContainer : CompositeDrawable
+    {
+        private const int duration = 200;
+        private const int margin = 10;
+        private const int width = 310;
+        private const int move_offset = 15;
+
+        public readonly Bindable<User> User = new Bindable<User>();
+
+        private readonly ContentContainer contentContainer;
```

Comment:
IIRC there is a `Content` property in the `Container` class. You probably should derive from `Container` and use it.

Reply:
Not required anymore, moved to GridContainer and removed the Content class